app/queries/app/global.ts

```diff
@@ -33,8 +37,47 @@ export const observeProfileLongPresTutorial = () => {
     if (!appDatabase) {
         return of$(false);
     }
-    return appDatabase.get<GlobalModel>(GLOBAL).query(Q.where('id', GLOBAL_IDENTIFIERS.PROFILE_LONG_PRESS_TUTORIAL), Q.take(1)).observe().pipe(
+    return queryGlobalValue(appDatabase, GLOBAL_IDENTIFIERS.PROFILE_LONG_PRESS_TUTORIAL).observe().pipe(
         switchMap((result) => (result.length ? result[0].observe() : of$(false))),
         switchMap((v) => of$(Boolean(v))),
     );
 };
+
+export const getLastAskedForReview = async () => {
+    const appDatabase = DatabaseManager.appDatabase?.database;
```

Comment:
here you should use `const {database} = DatabaseManager.getAppDatabaseAndOperator();`